browser-test/src/support/index.ts

```diff
@@ -205,3 +212,38 @@ export const validateAccessibility = async (page: Page) => {
 
   expect(results).toHaveNoA11yViolations()
 }
+
+/**
+ * Saves a screenshot to a file such as
+ * __snapshots__/test_file_name/name-of-the-test-1-snap.png
+ * If the screenshot already exists, compare the new screenshot with the
+ * existing screenshot, and save a pixel diff instead if the two don't match
+ */
+export const validateScreenshot = async (
+  page: Page,
+  pageScreenshotOptions?: PageScreenshotOptions,
+  matchImageSnapshotOptions?: MatchImageSnapshotOptions,
+) => {
+  // Do not make image snapshots when running locally
+  if (isLocalDevEnvironment()) {
```

Comment:
Looking at the implementation of isLocalDevEnvironment, it returns true for civiform:9000 and localhost:9999. 
civiform:9000 is used in run-browser-tests [here](https://github.com/civiform/civiform/blob/main/browser-test/bin/wait_for_server_start_and_run_tests.sh#L7)
localhost:9999 is used in [run-browser-tests-local](https://github.com/civiform/civiform/blob/main/bin/run-browser-tests-local#L7). 
And we need to disable it only for the second case but not the first one.

Reply:
Let's add another environment variable like DISABLE_SCREENSHOTS and set it in local run. We might need to set it in https://github.com/civiform/civiform/blob/main/bin/run-prober as well. So it will be a good to have a dedicated param that can be set to disable screenshots.